packages/hooks/use-combined-state/src/useCombinedState.tsx

```diff
@@ -1,34 +1,38 @@
-import { useMountedState } from '@spark-ui/use-mounted-state'
 import isEqual from 'lodash.isequal'
-import { useCallback, useLayoutEffect, useRef, useState } from 'react'
+import { SetStateAction, useCallback, useRef, useState } from 'react'
 
 /**
- * This hook must be used when a component have both a controlled and uncontrolled mode.
- * It will take care of updating the state value when controlled mode (prop) is updated.
+ * This hook must be used when a component has both a controlled and uncontrolled mode.
+ * It will take care of updating the state value when a controlled mode (prop) is updated.
  */
 export function useCombinedState<T>(
   controlledValue?: T,
   defaultValue?: T
```

Comment:
Great job @andresin87 do you think that would be possible to add the `onChange` callback like chakra does ? 🙏🏻 I think that could be really useful

Reply:
ok. I'll do today

Reply:
3rd argument

Reply:
DONE!